{{cookiecutter.project_dash_slug}}/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Base Image
+FROM python:3.6.6-alpine3.7
+
+# Best practice: if you build contains several layers you should order them
+# from the less frequently changed to the more frequently changed (caching)
+
+# Install tools required for the project
+# Best practice:
+# 1. do not install unnecessary packages, keep it small.
+# 2. keep an alphabetic order to avoid duplicates
+RUN apk add --no-cache --update \
+    build-base \
+    git \
+    libffi-dev \
+    linux-headers \
+    pcre-dev \
+    postgresql-client \
+    python3-dev
+
+# Run as non root user (best practice)
+ENV USERNAME=syapse
+ENV PROJECT_NAME={{cookiecutter.project_dash_slug}}
```

Comment:
I had originally done this in the Flask cookiecutter, where I named users and directories and stuff. But then I realized: the only thing running in this container is this project, and there's only one ever one user, so there's no need to give them special names: I just called the user "user" and put the package in `/opt/package`. It simplified the Dockerfile quite a lot and makes it the same everywhere.